modules/solid_mechanics/doc/content/source/materials/RadialReturnStressUpdate.md

```diff
@@ -112,6 +112,12 @@ allowed by substepping. A value of $1.0\cdot10^4$ will work for many cases. An e
 The `adaptive_substepping` parameter enables adaptive substepping, where the number of substeps is successively doubled until the
 return mapping model successfully converges or the doubled number of substeps exceeds the user specified maximum number of substeps.
 
+### Initializing Inelastic Strain Increment
+
+The scalar inelastic strain increment $\Delta p$ is normally intialized to zero and the constitutive model at the start of each timestep is intialized around a trial stress computed from a fully elastic strain increment predictor.  [!param](/Materials/PowerLawCreepStressUpdate/scale_strain_predictor)  allows a scaled value of the previous timesteps final $\Delta p$ to initialize the current timesteps intial $\Delta p$.  The predicted $\Delta p$ is removed from the elastic strain increment when computing the trial stress.  This new trial stress is then used to intialize the constitutive model around a state that is closer to the final state.  Initializing $\Delta p$ to a value close to its converged value will result in fewer nonlinear Newton iterations for the radial return mapping.
+
+The scaling factor [!param](/Materials/PowerLawCreepStressUpdate/scale_strain_predictor) ranges between zero and one.  Zero is the default which will not use the previous timestep's $\Delta p$.  The entire previous step's $\Delta p$ is used by setting `scale_strain_predictor`$=1$ which is usually too large of a guess for $\Delta p$ and will result in a constitutive model state where the stress is inside the yield surface.  A check is done to ensure the constitutive model state results in a stress inside the yield surface, and if it violates this condition, `scale_strain_predictor` is set to zero, reverting back to no $\Delta p$ predictor being used for the current timestep.  This capability works best when used with a [SimplePredictor.md] which predicts a solution state that should be close the final state.
```

Comment:
> The entire previous step's $\Delta p$ is used by setting `scale_strain_predictor`$=1$ which is usually too large of a guess for $\Delta p$ and will result in a constitutive model state where the stress is inside the yield surface.  A check is done to ensure the constitutive model state results in a stress inside the yield surface, and if it violates this condition, `scale_strain_predictor` is set to zero,

This is confusing:
1. A `scale_strain_predictor` of 1 is usually **too large** and will result in a stress **inside the yield surface**. (inside = bad?
2. You check to **ensure** the stress is **inside the yield surface. (inside = good?)
3. If the stress violates the condition (inside = bad again?) set scale_strain_predictor to zero (why not progressively cut it in half?)